documentation/APIs/openapi3-core.yaml

```diff
@@ -98,6 +108,186 @@ paths:
+      parameters:
+        - in: path
+          name: accountId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the account.
+        - in: query
+          name: slotIndex
+          schema:
+            type: integer
+          example: 1622568540
+          required: false
+          description: Check the account status for a specific slot index.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountResponse'
+
+  /api/core/v3/accounts/{accountId}/mana:
+    get:
+      tags:
+        - accounts
+      summary: Returns the potential and stored mana of the account. If slotIndex is not provided, the returned status would be current slot index. 
+      description: >-
+        Returns mana of an account.
+      parameters:
+        - in: path
+          name: accountId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the account.
+        - in: query
+          name: slotIndex
+          schema:
+            type: integer
+          example: 1622568540
+          required: false
+          description: Check the mana of an account for a specific slot index.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountManaResponse'
+
+  /api/core/v3/rewards/{outputId}:
+    get:
+      tags:
+        - rewards
+      summary: Returns the totally available rewards of the output ID. 
+      description: >-
+        Returns mana of an account.
+      parameters:
+        - in: path
+          name: outputId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the output.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountManaResponse'
+
+  /api/core/v3/committee:
+    get:
+      tags:
+        - committee
+      summary: Return the information of committee members. 
+      description: >-
+        TODO, performance factor, total pool stake
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountStakingResponse'
```

Comment:
This can return the information in the `Tracker.committeeStore` field, so 
- totalStake
- totalValidatorStake

and for each individual validator

- PoolStake
- ValidatorStake
- FixedCost